fault-proof/src/config.rs

```diff
@@ -214,6 +217,9 @@ impl ChallengerConfig {
             malicious_challenge_percentage: env::var("MALICIOUS_CHALLENGE_PERCENTAGE")
                 .unwrap_or("0.0".to_string())
                 .parse()?,
+            challenger_enable: env::var("CHALLENGER_ENABLE")
```

Comment:
personally I'd go with `DISABLE_MONITOR_ONLY` or something similar.

Having this `true` as a default is fine for now, but at some point we should negate the option and call it `ENABLE_MONITOR_ONLY` and have this `false` per default.